src/libraries/Microsoft.PowerFx.Core/Public/Types/RecordType.cs

```diff
@@ -96,5 +97,29 @@ public static RecordType Empty()
         {
             return new KnownRecordType();
         }
+
+        internal override void DefaultExpressionValue(StringBuilder sb)
+        {
+            var flag = true;
+
+            sb.Append("{");
+
+            foreach (var field in GetFieldTypes())
+            {
+                if (!flag)
+                {
+                    sb.Append(",");
+                }
+
+                flag = false;
+                
+                // !JYL! Check how to escape field name
```

Comment:
> // !JYL! Check how to escape field name [](http://example.com/codeflow?start=15&length=40)

todo

Reply:
The fact that we still pass means that we have a test gap in the .txts. 
Can we add a test that hits this failure, and then we can fix it.

Reply:
Added test cases for this.